lib/linguist/heuristics.yml

```diff
@@ -618,6 +618,12 @@ disambiguations:
     pattern: '^%(end|ctor|hook|group)\b'
   - language: Linker Script
     pattern: 'OUTPUT_ARCH\(|OUTPUT_FORMAT\(|SECTIONS'
+- extensions: ['.yaml']
+  rules:
+  - language: MiniYAML
+    pattern: '\t\S*.*:'
```

Comment:
Wouldn't this match a regular YAML file having a string that follows this pattern? Isn't there a `^` missing at the beginning? Something like `^\t+.*:` maybe? I'm not sure why the `\S*` is necessary.

```yaml
foo: "	bar:"
```

Reply:
Turns out this is valid MiniYAML
```salt
Grenade :
	ValidTargets : Ground, Trees
```
although I have never seen anyone do it.